spec/lib/query/query_supporters_spec.rb

```diff
@@ -282,7 +282,7 @@
           supporter_1 = force_create(:supporter, nonprofit_id: np.id, name: 'cacau borges', phone: '1234567890')
           supporter_2 = force_create(:supporter, nonprofit_id: np.id, name: 'Cacau Borges', phone: '(123)4567890')
 
-          expect(subject).to match_array([[supporter_1.id, supporter_2.id]])
+          expect(subject).to eq([[supporter_1.id, supporter_2.id].sort])
```

Comment:
Does this not work?

```suggestion
          expect(subject).to eq([[supporter_1.id, supporter_2.id]])
```

Reply:
I thought that it could become an intermitent test if for some reason their ids followed a different order

Reply:
I ran the tests a few times with your suggestion and none failed